packages/history-service/src/index.ts

```diff
@@ -34,9 +34,13 @@ export interface SharedHistoryService extends SharedFeatureService {
   readonly '1.1': FeatureServiceBinder<HistoryServiceV1>;
 }
 
+export interface RequiredFeatureServices {
```

Comment:
I like that we can type the required feature services now. I'm guessing you did the name change from `featureServices` to `requiredFeatureServices` because of the interface name (`RequiredFeatureServices`). I would prefer if we still called it `featureServices` though, despite the interface name. We don't really need the prefix there to discriminate them from some other feature services, don't you think?